src/stores/ResourceConfig/Store.ts

```diff
@@ -712,15 +744,36 @@ const getInitialState = (
         }
 
         if (editWorkflow && liveSpecIds.length > 0) {
-            const { data, error } = draftId
-                ? await getDraftSpecsByDraftId(draftId, entityType)
-                : await getLiveSpecsByLiveSpecId(liveSpecIds[0], entityType);
+            const { data: liveSpecs, error: liveSpecError } =
+                await getLiveSpecsByLiveSpecId(liveSpecIds[0], entityType);
 
-            if (error) {
+            if (liveSpecError) {
                 setHydrationErrorsExist(true);
-            } else if (data && data.length > 0) {
-                const { prefillResourceConfig } = get();
-                prefillResourceConfig(sortBindings(data[0].spec.bindings));
+            } else if (liveSpecs && liveSpecs.length > 0) {
+                const { prefillBackfilledCollections, prefillResourceConfig } =
+                    get();
+
+                let selectedSpec:
+                    | LiveSpecsExtQuery
+                    | DraftSpecsExtQuery_ByDraftId = liveSpecs[0];
+
+                if (draftId) {
+                    const { data: draftSpecs, error: draftSpecError } =
+                        await getDraftSpecsByDraftId(draftId, entityType);
+
+                    if (draftSpecError) {
+                        setHydrationErrorsExist(true);
```

Comment:
How does this work? Cause this is totally different logic now.

Even if there is an error the prefill of resource config will happen.

Reply:
The live spec is used as a fallback... which is new behavior. I can fix this.

Reply:
This should be resolved.